core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.css

```diff
@@ -431,14 +433,49 @@
   word-break: break-word;
 }
 
+.save-progress-menu .uid-box .uid-copy .uid-btn-container {
+  margin-left: 2%;
+  position: relative;
+  width: 20%;
+}
+
 .save-progress-menu .uid-box .uid-copy .uid-copy-btn {
   box-shadow: 1px 1px 2px 0 rgba(0, 100, 92, 0.8);
   color: #00645c;
   font-family: Roboto, sans-serif;
   font-size: 14px;
   font-weight: 700;
   margin: 0 auto;
-  width: 20%;
+  position: relative;
+  width: 100%;
+}
+
+.save-progress-menu .uid-box .uid-copy .uid-btn-container .uid-btn-message-box {
+  background-color: #00645c;
+  border-radius: 4px;
+  color: white;
```

Comment:
Please use hex color code here and elsewhere.

Reply:
changed to the hex code for the colors